recipe/meta.yaml

```diff
@@ -14,15 +14,17 @@ source:
     - nss-3.36.1-standalone-1.patch
 
 build:
-  number: 2
+  number: 1002
   skip: true  # [win]
 
 requirements:
   build:
-    - autoconf
-    - cmake
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - autoconf
+    - cmake
+    - pkgconfig
```

Comment:
Should be `pkg-config`. The other is a Python package.

Reply:
Thanks to point it out! :)